GKE_docker_image_tool/build_docker_image.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description:
+# bash build_docker_image.sh MODE=stable
+# bash build_docker_image.sh MODE=nightly
+# bash build_docker_image.sh MODE=head
+
+# bash build_docker_image.sh MODE=stable PROJECT=cloud-tpu-multipod-dev
+# bash build_docker_image.sh MODE=stable JAX_VERSION=0.4.13 PROJECT=cloud-tpu-multipod-dev
+# bash build_docker_image.sh MODE=stable JAX_VERSION=0.4.13 PROJECT=cloud-tpu-multipod-dev IMAGE_PREFIX=jax0.4.13
+
+
+# Enable "exit immediately if any command fails" option
+set -e
+
+export PROJECT=tpu-prod-env-multipod
```

Comment:
Can this one come from the environment? Something like:
https://source.corp.google.com/piper///depot/google3/experimental/users/vbarr/multipod/xpk/xpk.py;l=231-248?q=xpk

Reply:
Done